lib/node/rpc.js

```diff
@@ -2152,16 +2152,21 @@ class RPC extends RPCBase {
   }
 
   async estimateFee(args, help) {
-    if (help || args.length !== 1)
-      throw new RPCError(errs.MISC_ERROR, 'estimatefee nblocks');
+    if (help || args.length < 1 || args.length > 2)
+      throw new RPCError(errs.MISC_ERROR, 'estimatefee nblocks (covenantType)');
```

Comment:
Using `( )` means that it is an optional argument. This means that the args length checker needs to be updated. If you want it to be a non optional argument, then you can simplify into `(help || args.length !== 2)`

See:

https://github.com/handshake-org/hsd/blob/7c4abc2938e37c4db8e8800f52d7386e2ddace38/lib/node/rpc.js#L719

Also can you add `"` around the arguments like: `"nblocks" ( "covenantType" )`

Reply:
fixed!